src/crystal/system/unix/login.cr

```diff
@@ -0,0 +1,11 @@
+require "c/unistd"
+
+module Crystal::System
+  def self.login
+    if login = LibC.getlogin
+      String.new(login)
+    else
+      nil
```

Comment:
If a condition is true then return something, otherwise nothing, aka `nil` in crystal terms. Adding `else nil` is just noise.